src/KubernetesClient/Kubernetes.ConfigInit.cs

```diff
@@ -243,6 +246,10 @@ private void CreateHttpClient(DelegatingHandler[] handlers, KubernetesClientConf
 
             AppendDelegatingHandler<WatcherDelegatingHandler>();
             HttpClient = new HttpClient(FirstMessageHandler, false);
+
+#if NET5_0
+            HttpClient.DefaultRequestVersion = HttpVersion.Version20;
```

Comment:
This does NOT work, 
see <https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.defaultrequestversion?view=net-5.0>

> The DefaultRequestVersion property doesn't apply to the **SendAsync** method. The HttpRequestMessage parameter passed as the argument to the SendAsync method has its own Version property that controls the HTTP version used for the request.

Unfortunately, our generated code use `SendAsync` LOL


Here is my test (Should find a way to do automation)

1. create a AKS (1.19)
1. disable tcp keepalive
1. start watch example

got a [R] from remote side after 5 min and then `watch` will not show any event from `kubectl run` 

```
02:42:58.973029 IP 10.x.x.x.49746 > 52.x.x.x.443: Flags [.], ack 28897, win 492, options [nop,nop,TS val 255369207 ecr 3243887534], length 0
02:47:48.004159 IP 52.x.x.x.443 > 10.x.x.x.49746: Flags [R], seq 660228079, win 0, length 0
```